java/src/test/java/com/saucelabs/simplesauce/BaseConfigurationTest.java

```diff
@@ -1,28 +1,37 @@
 package com.saucelabs.simplesauce;
 
 import org.junit.Before;
+import org.junit.Rule;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+import org.openqa.selenium.remote.RemoteWebDriver;
 
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
-class BaseConfigurationTest {
+public class BaseConfigurationTest {
     protected SauceSession sauce;
     protected SauceOptions sauceOptions;
-    private SauceRemoteDriver dummyRemoteDriver;
-    private EnvironmentManager dummyEnvironmentManager;
+    protected RemoteWebDriver dummyRemoteDriver = mock(RemoteWebDriver.class);
+    protected EnvironmentManager dummyEnvironmentManager;
+
+    @Rule
+    public MockitoRule initRule = MockitoJUnit.rule();
 
     @Before
     public void setUp() {
-        dummyRemoteDriver = mock(SauceRemoteDriver.class);
+        dummyRemoteDriver = mock(RemoteWebDriver.class);
         dummyEnvironmentManager = mock(EnvironmentManager.class);
         sauceOptions = new SauceOptions();
 
-        sauce = new SauceSession(dummyRemoteDriver, dummyEnvironmentManager);
         when(dummyEnvironmentManager.getEnvironmentVariable("SAUCE_USERNAME")).thenReturn("test-name");
         when(dummyEnvironmentManager.getEnvironmentVariable("SAUCE_ACCESS_KEY")).thenReturn("accessKey");
     }
 
-    SauceSession instantiateSauceSession() {
-        return new SauceSession(sauceOptions, dummyRemoteDriver, dummyEnvironmentManager);
+    void startSauceSession() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/SauceLabs/simple_sauce/pullRequest?prid=4730723)